core/jazz-ui/src/app/secondary-components/create-service/internal/create-service.component.ts

```diff
@@ -1156,7 +1156,7 @@ blurRegion(){
   //   this.git_private =  gitPrivate.checked;
```

Comment:
remove commented code in all the places

Reply:
This is on an internal file and is currently irrelevant to oss Jazz

Reply:
same comment as before

Reply:
removing commented code